2020/day3.clj

```diff
@@ -44,31 +51,45 @@
         ...
         ".#..#...#.#"]
```

Comment:
Does `sample-input` convey what this is?

`input` as a parameter is a HUGE code smell, it should likely never be in your code

Reply:
> Does `sample-input` convey what this is?
> 
> `input` as a parameter is a HUGE code smell, it should likely never be in your code

rewrote with "sample-grid"